### ISSUE TITLE
Reduce the number of write buffers for infrequently used column families

### ISSUE THREAD

**Issue by git-hulk** [apache/kvrocks#2193]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/kvrocks/issues) and found no similar issues.

### Motivation

Currently, all column families are shared the same number of write buffers
which is unnecessary for infrequently used column families including `pubsub`,
`propagte`, and `zset_score`. We can keep them to a const value like 1 to reduce
the memory usage in some scenarios.

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

**Comment by LiuYuHui** [comment]
Please assign it to me, thanks a lot :D

**Comment by git-hulk** [comment]
@LiuYuHui Thank you!

**Comment by git-hulk** [comment]
Hi @LiuYuHui Are you still working on this issue?

**Comment by LiuYuHui** [comment]
Hi @git-hulk, I have had no extra time bandwidth recently, please feel free to assign it to other people.

**Comment by git-hulk** [comment]
@LiuYuHui Thanks for your quick response. Welcome back while you get time.

**Comment by jjz921024** [comment]
Hi @git-hulk , May i question? How do we implement this feature? 

1. For each column family add a config, specifying its number of write buffers. eg: `pubsub.max_write_buffer_number`、`propagte.max_write_buffer_number`、`zset_score.max_write_buffer_number` ...
2. Add a `lite` mode config, When this config is set to `true`, create a new `lite_table_opts` for infrequently used column families. like:

```java
rocksdb::BlockBasedTableOptions lite_table_opts = InitTableOptions();
rocksdb::ColumnFamilyOptions lite_opts(subkey_opts);
lite_opts.write_buffer_size = 32 * MiB;
lite_opts.max_write_buffer_number = 0;

// replace subkey_opts with lite_opts 
column_families.emplace_back(kZSetScoreColumnFamilyName, lite_opts);
column_families.emplace_back(kPubSubColumnFamilyName, lite_opts);
column_families.emplace_back(kPropagateColumnFamilyName, lite_opts);
column_families.emplace_back(kStreamColumnFamilyName, lite_opts);```

**Comment by git-hulk** [comment]
@jjz921024 Yes, I'm good with this implementation. Would you like to submit a PR?

**Comment by jjz921024** [comment]
Yes, I want to try

**Comment by git-hulk** [comment]
@jjz921024 Assigned, thank you!
